e2e_test/mock.go

```
@@ -0,0 +1,98 @@
+		mb.handleDefault(w, r)
+	}
+}
+
+func (mb *MockBackend) handleEchoBody(w http.ResponseWriter, r *http.Request) {
+	body, err := io.ReadAll(r.Body)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
+	w.Write(body)
```

golangci-lint

Error return value of `w.Write` is not checked

(errcheck)